web/app/views/layouts/embedded_app.html.erb

```diff
@@ -13,27 +13,16 @@
       <%= javascript_include_tag 'application', "data-turbolinks-track" => true %>
     <% end %>
     <%= csrf_meta_tags %>
+
+    <%= javascript_include_tag 'https://cdn.shopify.com/shopifycloud/app-bridge.js', data: {
+      'api-key': ShopifyApp.configuration.api_key,
```

Comment:
can this be `api-key` or does it need to be `shopify-api-key` ?
https://shopify.dev/docs/api/app-bridge-library#getting-started

Reply:
Good question! When passing in the key in the dataset, `data-api-key` works, that's what we use [in the remix template](https://github.com/Shopify/shopify-app-js/blob/main/packages/apps/shopify-app-remix/src/react/components/AppProvider/AppProvider.tsx#L116) as well.

I've tested this in a real app and it was able to make authenticated `fetch` requests, so app bridge was loading properly.